metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -803,28 +858,44 @@ def quickstart(
     )
 
 
+def get_docker_compose_base_url(version_tag: Optional[str]) -> str:
```

Comment:
@pedro93 
>If unstable, where does the code pick up the compose files from?
This method decided where will the files picked up from.

Reply:
What if we want an unstable version from a different compose base path? I.e: To test HEAD images with compose PR changes?